baseapp/setup.cfg

```diff
@@ -36,7 +36,7 @@ install_requires =
     django-ipware >= 7.0.1
     python-ipware >= 3.0.0
     requests >= 2.32.3
-
+    stripe >= 11.5.0
```

Comment:
rebase please and fix this, this should be under `payments` on `extras_require` bellow